README.md

```diff
@@ -127,9 +127,9 @@ There is a `build.bat`/`build.sh` file that creates a necessary generated file n
 #### Windows
 Prerequisites:
 
- * .NET Framework 3.5 (This is a windows feature installation).
- * .NET Framework 4+
- * Visual Studio is helpful for working on source.
+ * .NET Framework 4.8+
+ * .NET SDK 6.0+

```

Comment:
@TheCakeIsNaOH can you elaborate on this requirement?  While Chocolatey.Cake.Recipe can build .NET SDK projects, the installation of it is only required when targeting that framework.  chocolatey/choco still calls the `Build.Run()` rather than the `Build.RunDotNet()` method.

Reply:
As per our discussion, it is not required on Windows, but only on Linux due to the dotnet tools required.